frontend/src/App.js

```diff
@@ -12,7 +12,8 @@ class App extends Component {
         token: '',
         username: '',
         fName: '',
-        lName: ''
+        lName: '',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/StevenMonty/CS1530-Final/pullRequest?prid=7283612)
```suggestion
        lName: "",
```